app/editor/src/features/admin/users/UserFilter.tsx

```diff
@@ -1,5 +1,5 @@
 import { IconButton, Select } from 'components/form';
-import { UserStatusName } from 'hooks';
+import { UserRoleName, UserStatusName } from 'hooks';
```

Comment:
Roles can be extracted from the `useLookups()` redux hook.  You may need to update the hook controller.  Don't hard-code roles.  They come from the API.

Reply:
Updated, thanks